Modules/Microsoft365DSC/DSCResources/MSFT_AADAuthenticationMethodPolicyAuthenticator/MSFT_AADAuthenticationMethodPolicyAuthenticator.schema.mof

```diff
@@ -0,0 +1,51 @@
+[ClassVersion("1.0.0")]
+class MSFT_MicrosoftGraphMicrosoftAuthenticatorFeatureSettings
+{
+    [Write, Description("Determines whether users will be able to approve push notifications on other Microsoft applications such as Outlook Mobile."), EmbeddedInstance("MSFT_MicrosoftGraphAuthenticationMethodFeatureConfiguration")] String CompanionAppAllowedState;
+    [Write, Description("Determines whether the user's Authenticator app will show them the client app they are signing into."), EmbeddedInstance("MSFT_MicrosoftGraphAuthenticationMethodFeatureConfiguration")] String DisplayAppInformationRequiredState;
+    [Write, Description("Determines whether the user's Authenticator app will show them the geographic location of where the authentication request originated from."), EmbeddedInstance("MSFT_MicrosoftGraphAuthenticationMethodFeatureConfiguration")] String DisplayLocationInformationRequiredState;
+    [Write, Description("Specifies whether the user needs to enter a number in the Authenticator app from the login screen to complete their login. Value is ignored for phone sign-in notifications."), EmbeddedInstance("MSFT_MicrosoftGraphAuthenticationMethodFeatureConfiguration")] String NumberMatchingRequiredState;
+};
+[ClassVersion("1.0.0")]
+class MSFT_MicrosoftGraphAuthenticationMethodFeatureConfiguration
+{
+    [Write, Description("A single entity that's excluded from using this feature."), EmbeddedInstance("MSFT_MicrosoftGraphFeatureTarget")] String ExcludeTarget;
+    [Write, Description("A single entity that's allowed to use this feature."), EmbeddedInstance("MSFT_MicrosoftGraphFeatureTarget")] String IncludeTarget;
+    [Write, Description("Enable or disable the feature. Possible values are: default, enabled, disabled, unknownFutureValue. The default value is used when the configuration hasn't been explicitly set and uses the default behavior of Azure AD for the setting. The default value is disabled."), ValueMap{"default","enabled","disabled","unknownFutureValue"}, Values{"default","enabled","disabled","unknownFutureValue"}] String State;
+};
+[ClassVersion("1.0.0")]
+class MSFT_MicrosoftGraphFeatureTarget
+{
+    [Write, Description("The ID of the entity that's targeted in the include or exclude rule or all_users to target all users.")] String Id;
+    [Write, Description("The kind of entity that's targeted. The possible values are: group, administrativeUnit, role, unknownFutureValue."), ValueMap{"group","administrativeUnit","role","unknownFutureValue"}, Values{"group","administrativeUnit","role","unknownFutureValue"}] String TargetType;
+};
+[ClassVersion("1.0.0")]
+class MSFT_MicrosoftGraphExcludeTarget2
```

Comment:
To avoid conflicts across resources in the future, we took the decision to always rename these CIMInstances as:

MSFT_resourceNameCIMInstance

Please review every schema.mof and module files in this Pull Request to adhere to this naming standard. Thanks